test/unit/TransmuteDPOS.spec.js

```diff
@@ -67,7 +65,7 @@ contract('TransmuteDPOS', accounts => {
       assert.equal(42, totalBondedAmount);
     });
 
-    it("should register a Provider's parameters", async () => {
+    it('should register a Provider\'s parameters', async () => {
       assert.equal(PROVIDER_UNREGISTERED, await tdpos.providerStatus.call(accounts[1]));
       await approveBondProvider(10, 20, 2, 35, 1, accounts[1]);
```

Comment:
these numbers: 10, 20, 2, 35, 1, should be constants or variables with descriptive names.